### PR TITLE
Cleanup ASP.Net integration steps

### PR DESCRIPTION
**Pull request by robertnicolo-okta**
## Description:
Clean up of the ASP.Net integration steps. Specifics on which pages needed what changes can be found in [OKTA-417607](https://oktainc.atlassian.net/browse/OKTA-417607).

### Resolves:

* [OKTA-417607](https://oktainc.atlassian.net/browse/OKTA-417607)

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-cases/oie-embedded-sdk-use-case-self-reg/aspnet/integrationsteps.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-### Step 1: Create a sign-up link for new users
+### Step 1: Click sign-up link
```
Comment (chrisdmills-okta) [inline packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-cases/oie-embedded-sdk-use-case-self-reg/aspnet/integrationsteps.md:1]: We agreed in the standup meeting where we discussed this that we'd remove the word "Step" from the step headings, didn't we? So for example this line would just be `### 1: Click sign-up link`

File: packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-cases/oie-embedded-sdk-use-case-self-reg/aspnet/integrationsteps.md [hunk 2]
```diff
@@ -190,14 +190,14 @@ var enrollAuthenticatorOptions = new EnrollAuthenticatorOptions
 var enrollResponse = await idxAuthClient.EnrollAuthenticatorAsync(enrollAuthenticatorOptions, (IIdxContext)Session["IdxContext"]);
 ```
 
-### Step 12: Update the Email Factor Verification email template (optional)
+### Step 12 Optional: Update sent email template
```
Comment (chrisdmills-okta) [inline packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-cases/oie-embedded-sdk-use-case-self-reg/aspnet/integrationsteps.md:193]: Put "Optional" in parens (i.e. `(Optional)`) for consistency with other guides.

Comment (chrisdmills-okta) [reply packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-cases/oie-embedded-sdk-use-case-self-reg/aspnet/integrationsteps.md:193]: Also check other instances of "Optional"

File: packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-cases/oie-embedded-sdk-use-case-self-reg/aspnet/integrationsteps.md [hunk 3]
```diff
@@ -239,7 +239,7 @@ var idxAuthClient = new IdxClient(null);
 var authnResponse = await idxAuthClient.VerifyAuthenticatorAsync(verifyAuthenticatorOptions, (IIdxContext)Session["idxContext"]);
 ```
 
-### Step 16: Handle response from VerifyAuthenticatorAsync
+### Step 16: Handle the submit response
```
Comment (chrisdmills-okta) [inline packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-cases/oie-embedded-sdk-use-case-self-reg/aspnet/integrationsteps.md:242]: In the corresponding diagram, the steps are mis-numbered — after 15 goes straight to 17. 

Also, what about Step 18 (17?) — Show the remaining list of authenticators ?

File: packages/@okta/vuepress-site/docs/guides/oie-embedded-widget-use-cases/oie-embedded-widget-use-case-load/aspnet/integrationsteps.md [hunk 4]
```diff
@@ -1,4 +1,4 @@
-### Step 1: Call StartWidgetSigninAsync
+### Step 1: Navigate to sign-in page
 
 #### Summary
```
Comment (chrisdmills-okta) [inline packages/@okta/vuepress-site/docs/guides/oie-embedded-widget-use-cases/oie-embedded-widget-use-case-load/aspnet/integrationsteps.md:3]: This step has "Summary" and "Code sample" subheadings, which I've not seen in any other use cases. I'd delete them for consistency.

File: packages/@okta/vuepress-site/docs/guides/oie-embedded-widget-use-cases/oie-embedded-widget-use-case-sign-in-soc-idp/aspnet/integrationsteps.md [hunk 5]
```diff
@@ -39,15 +40,25 @@ public async Task<ActionResult> Callback(string state = null, string interaction
 }
 ```
 
-### Step 5: Call RedeemInteractionCodeAsync to get tokens
+### Step 5: Get tokens
```
Comment (chrisdmills-okta) [inline packages/@okta/vuepress-site/docs/guides/oie-embedded-widget-use-cases/oie-embedded-widget-use-case-sign-in-soc-idp/aspnet/integrationsteps.md:43]: This is "Get the tokens" in the diagram.

**Review by chrisdmills-okta — CHANGES_REQUESTED** [review]
Good work @robertnicolo-okta ! Not too many comments here.

File: packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-cases/oie-embedded-sdk-use-case-self-reg/aspnet/integrationsteps.md (hunk 1 above)
Comment (robertnicolo-okta) [reply packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-cases/oie-embedded-sdk-use-case-self-reg/aspnet/integrationsteps.md:1]: "Step" text removed.

File: packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-cases/oie-embedded-sdk-use-case-self-reg/aspnet/integrationsteps.md (hunk 2 above)
Comment (robertnicolo-okta) [reply packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-cases/oie-embedded-sdk-use-case-self-reg/aspnet/integrationsteps.md:193]: Added parentheses and also checked the other optionals I changed and ensured consistency.

File: packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-cases/oie-embedded-sdk-use-case-self-reg/aspnet/integrationsteps.md (hunk 3 above)
Comment (robertnicolo-okta) [reply packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-cases/oie-embedded-sdk-use-case-self-reg/aspnet/integrationsteps.md:242]: Fixed the numbering in the diagram. steps 15, 16, 17 are now aligned with the doc.  Step 18 is in the next diagram.

File: packages/@okta/vuepress-site/docs/guides/oie-embedded-widget-use-cases/oie-embedded-widget-use-case-load/aspnet/integrationsteps.md (hunk 4 above)
Comment (robertnicolo-okta) [reply packages/@okta/vuepress-site/docs/guides/oie-embedded-widget-use-cases/oie-embedded-widget-use-case-load/aspnet/integrationsteps.md:3]: subheadings removed.

File: packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-cases/oie-embedded-sdk-use-case-self-reg/aspnet/integrationsteps.md [hunk 6]
```diff
@@ -190,14 +190,14 @@ var enrollAuthenticatorOptions = new EnrollAuthenticatorOptions
 var enrollResponse = await idxAuthClient.EnrollAuthenticatorAsync(enrollAuthenticatorOptions, (IIdxContext)Session["IdxContext"]);
 ```
 
-### Step 12: Update the Email Factor Verification email template (optional)
+### 12: (Optional) Update sent email template
```
Comment (chrisdmills-okta) [inline packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-cases/oie-embedded-sdk-use-case-self-reg/aspnet/integrationsteps.md:193]: Nitpick — I think the colon should be put after `(Optional)`, as it's not part of the step, but more part of the meta data about the step (i.e. the step number and whether it's optional both sounds like metadata)

File: packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-cases/oie-embedded-sdk-use-case-self-reg/aspnet/integrationsteps.md [hunk 7]
```diff
@@ -273,13 +273,13 @@ The remaining authenticator should display the phone factor to the user. Since t
 
 The user can either enroll in the phone factor or skip the phone factor. Your code should handle both scenarios that will be described in the following steps.
 
-### Step 18a, Option 1, Enroll and verify the phone authenticator
+### 18a, Option 1, Select phone authenticator
```
Comment (chrisdmills-okta) [inline packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-cases/oie-embedded-sdk-use-case-self-reg/aspnet/integrationsteps.md:276]: In this one you've got the structure `number, option number, step heading`, and in the one below you've got the structure `number: option number: step heading`. This needs to be consistent, and I think we should do `number, option number: step heading` — keep the number and option the same as it is written in the diagrams, but keep the meta data and step heading separated by a colon.

File: packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-cases/oie-embedded-sdk-use-case-self-reg/aspnet/integrationsteps.md [hunk 8]
```diff
@@ -402,7 +402,7 @@ switch (authnResponse.AuthenticationStatus)
 }
 ```
 
-### Step 19, Option 2: Call SkipAuthenticatorSelectionAsync
+### 19, Option 2: Skip phone enrollment
```
Comment (chrisdmills-okta) [inline packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-cases/oie-embedded-sdk-use-case-self-reg/aspnet/integrationsteps.md:405]: "19" -> "19a"

File: packages/@okta/vuepress-site/docs/guides/oie-embedded-widget-use-cases/oie-embedded-widget-use-case-sign-in-soc-idp/aspnet/integrationsteps.md [hunk 9]
```diff
@@ -39,15 +40,25 @@ public async Task<ActionResult> Callback(string state = null, string interaction
...
+
+Persist the tokens in session for future use. The following code below from the sample application uses
+`IAuthenticationManager` from `Microsoft.Owin.Security` to persist these tokens in session.
+
+```csharp
+ClaimsIdentity identity = await AuthenticationHelper.GetIdentityFromTokenResponseAsync(_idxClient.Configuration, tokens);
+_authenticationManager.SignIn(new AuthenticationProperties { IsPersistent = false }, identity);
+```
+
+### 7: (Optional) Get user profile information
```
Comment (chrisdmills-okta) [inline packages/@okta/vuepress-site/docs/guides/oie-embedded-widget-use-cases/oie-embedded-widget-use-case-sign-in-soc-idp/aspnet/integrationsteps.md:61]: Again, put the colon after "(Optional)"

**Review by chrisdmills-okta — APPROVED** [review]
@robertnicolo-okta looking good mate, thanks! I had a couple more minor nitpicks, but I don't think I need to see this again after you've fixed those, so I've just set it to approved instead of request changes.

**Pull request merged**